<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color:qlineargradient(spread:pad, x1:0.091, y1:0.101636, x2:0.991379, y2:0.977, stop:0 rgba(209, 107, 165, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1920</x>
     <y>1100</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>381</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>70</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="LogOut">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#LogOut{
	Background-color:rgba(85,98,112,255);
	color:rgba(255,255,255,200);
	border-radius:5px;
} 
QPushButton#LogOut:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(254, 81, 171, 0.8);
	background-position:calc(100% - 10px)center;
}
QPushButton#LogOut:hover{
	background-color:rgba(254, 81, 171, 0.8)
}</string>
   </property>
   <property name="text">
    <string>LogOut</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>230</y>
     <width>800</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 20px;
background-color:rgb(255,255,255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Addpatient">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#Addpatient{
	Background-color:rgba(85,98,112,255);
	color:rgba(255,255,255,200);
	border-radius:5px;
} 
QPushButton#Addpatient:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(254, 81, 171, 0.8);
	background-position:calc(100% - 10px)center;
}
QPushButton#Addpatient:hover{
	background-color:rgba(254, 81, 171, 0.8)
}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletepatient">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#deletepatient{
	Background-color:rgba(85,98,112,255);
	color:rgba(255,255,255,200);
	border-radius:5px;
} 
QPushButton#deletepatient:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(254, 81, 171, 0.8);
	background-position:calc(100% - 10px)center;
}
QPushButton#deletepatient:hover{
	background-color:rgba(254, 81, 171, 0.8)
}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>12 Calibri</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
font: 16pt bold &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Low</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>70</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>80</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>12 Calibri</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
font: 16pt bold &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>260</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>270</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>12 Calibri</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
font: 16pt bold &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
